stock_picking_batch_extended/views/report_batch_picking.xml

```diff
@@ -14,128 +14,124 @@
         >stock_picking_batch_extended.report_batch_picking</field>
     </record>
 
-    <template id="report_batch_picking">
-        <t t-call="web.html_container">
-            <t t-call="web.external_layout">
-                <div class="page">
-                    <div class="row mt32 mb32">
-                        <div class="col-3">
-                            <strong>Date:</strong>
-                            <span t-esc="now()" t-options="{'widget': 'datetime'}" />
-                        </div>
-                        <div class="col-3">
-                            <strong>Printed by:</strong>
-                            <span t-field="user.name" />
-                        </div>
+    <template id="report_batch_picking_document">
```

Comment:
Why this has a different level of indentation?

Reply:
I don't understand what it refers to. The template has the same level of indentation as the previous record `action_report_batch_picking` or the following template `report_batch_picking`

If you are referring to the report_name or report_file fields, those are line breaks introduced by pre-commit.